back/scripts/communities/loaders/sirene.py

```diff
@@ -15,8 +13,8 @@ class SireneLoader:
 
     def __init__(self, config):
         base_path = get_project_base_path()
-        data_folder = Path(base_path / config["path"])
-        self.data = pd.read_csv(data_folder / config["filename"], usecols=config["columns"])
+        payload_path = base_path / config["scrapped_data_file"]
+        self.data = pd.read_csv(payload_path, usecols=config["columns"])
```

Comment:
nits: don't really need `base_path` and variable name isn't representative
```suggestion
        filepath = get_project_base_path() / config["scrapped_data_file"]
        self.data = pd.read_csv(filepath, usecols=config["columns"])
```